README.md

```diff
@@ -21,7 +16,114 @@ detect components (the concept of component is taken from Odo and its definition
 
 ## Usage
 
-- [Go library & CLI](go/README.md#Usage)
+### CLI
+
+The Go CLI can be built with the below command:
+
+```bash
+# inside the go/ dir
+$ go build alizer.go
+```
+
+```shell
+$ ./alizer analyze <path>
+$ ./alizer component <path>
+$ ./alizer devfile <path>
+```
+
+### CLI Arguments
+
+#### --log
+
+Sets the logging level of the CLI. The arg accepts only 3 values [`debug`, `info`, `warning`]. The default value is `warning` and the logging level is `ErrorLevel`. Example usage
+
+```shell
+$ ./alizer analyze --log debug <path>
+```
+
```

Comment:
* for devfile command we have the `--registry -r` arg which specifies the registry we will use.
* for the component command we have  `port-detection -p` ("--port detection strategy to use when detecting a port. Currently supported strategies are 'docker', 'compose' and 'source'. You can pass more strategies at the same time. They will be executed in order. By default Alizer will execute docker, compose and source") as it mentioned in the code.